man/networkd.conf.xml

```diff
@@ -172,6 +172,30 @@ DUIDRawData=00:00:ab:11:f9:2a:c2:77:29:f9:5c:00</programlisting>
     </variablelist>
   </refsect1>
 
+  <refsect1>
+      <title>[MPTCP] Section Options</title>
+      <para>The [MPTCP] section accepts the following keys. </para>
+
+      <variablelist>
+      <varlistentry>
+        <term><varname>LimitSubflows=</varname></term>
+        <listitem>
+          <para>Specifies if additional subflow creation is allowed by MPTCP limits, the endpoint will be used as the source address to
+          create an additional subflow after that the MPTCP connection is established. Takes an unsigned integer in the range 1..8.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>LimitAddress=</varname></term>
```

Comment:
Hmm, this is names as if you#d configure an address here. Maybe call this AddressMax? And then also SubflowsMax?

or AddressLimit=? or SubflowsLimis?